File: JAVA/Memory.java
public class Memory extends Hardware{

    String frekuensi;
    String size;
    String cuda;
    
    Memory(){

    }


    Memory(String frekuensi, String size, String cuda){
        this.frekuensi = frekuensi;
        this.size = size;
        this.cuda = cuda;
    }

    void setFrekuensi(String frekuensi){
        this.frekuensi = frekuensi;
    }

    void setSizes(String size){
        this.size = size;
    }

    void setCuda(String cuda){
        this.cuda = cuda;
    }

    String getFrekuensi(){
        return this.frekuensi;
    }

    String getSize(){
        return this.size;
    }

    String getCuda(){
        return this.cuda;
    }

}

File: C++/Memory.cpp
#include<iostream>
#include<string>
#include "Hardware.cpp"


using namespace std;



class Memory :  public Hardware{

    private:
    int frekuensi;
    int size;
    string cuda;
    

    public :
    Memory(){

    }


    Memory(int fr, int sz, string ca){
        frekuensi = fr;
        size = sz;
        cuda = ca;
    }

    void setFrekuensi(int fr){
        frekuensi = fr;
    }

    void setSizes(int sz){
        size = sz;
    }

    void setCuda(string ca){
        cuda = ca;
    }

    int getFrekuensi(){
        return frekuensi;
    }

    int getSize(){
        return size;
    }

    string getCuda(){
        return cuda;
    }


    ~Memory(){
        
    }
};
